great_expectations/cli/datasource.py

```diff
@@ -625,6 +636,10 @@ def credentials_snippet(self) -> str:
 private_key_path = "YOUR_KEY_PATH"  # Path to the private key used for authentication
 private_key_passphrase = ""   # Passphrase for the private key used for authentication (optional -- leave blank for none)"""
 
+        snippet += '''\n
+# A table that you would like to add initially as a Data Asset
+table_name = "YOUR_TABLE_NAME"'''
+
```

Comment:
Just marking that this snippet is for `snowflake` only

Reply:
Correct!